airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -214,6 +214,10 @@ def name(self) -> str:
         if stream_name.endswith("Stream"):
             stream_name = stream_name[: -len("Stream")]
         return stream_name
+    
+    @abstractmethod
```

Comment:
there is something wrong here, we declare method abstract (signatures don't match btw) and implement only in one stream, other streams can't be created because they abstract classes now.